examples/tests/valid/container_types.w

```diff
@@ -16,6 +17,7 @@ assert(arr1.length == 3);
 assert(arr2.at(1) == "2");
 
 //Map tests
+let empty_map = Map<num>{};
```

Comment:
IMO `{}` should be a compiler error, both because:
- We don't know what type the values are (`Map<str>`? `Map<num>`?)
- It looks like it could also be a set
@staycoolcall911 @yoav-steinberg thoughts?

Reply:
hmmm... good point @Chriscbr. I think you're correct. In any case, implicit map init is currently not supported, so I'm removing it from this PR.

Reply:
oh, right, we don't know what type it is. cancel my comment, this should be an error.